programs/pre-authorized-debit-v1/src/errors.rs

```diff
@@ -12,8 +12,8 @@ pub enum CustomProgramError {
     #[msg("Last debited cycle is after current debited cycle (invalid state)")]
     LastDebitedCycleBeforeCurrentCycle, // 6002 or 0x1772
 
-    #[msg("Cannot change pre-authorization variant")]
-    CannotChangePreAuthorizationVariant, // 6003 or 0x1773
+    #[msg("Invalid timestamp value provided")]
+    InvalidTimestamp, // 6003 or 0x1773
```

Comment:
This was not used anywhere

Reply:
Oh yeah that was an old error for an IX I abandoned